railties/lib/rails/generators/rails/plugin/plugin_generator.rb

```diff
@@ -66,6 +66,16 @@ def gitignore
       template "gitignore", ".gitignore"
     end
 
+    def cifiles
```

Comment:
Very minor, and you might have a reason to do this, but could we separate CI and files with a underscore?

```suggestion
    def ci_files
```

it took me a while to understand it at first.

Reply:
The app generator calls it `cifiles` so I'd rather keep it consistent.